lib/std/tz.zig

```diff
@@ -11,7 +11,7 @@ pub const Timetype = struct {
     flags: u8,
     name_data: [6:0]u8,
 
-    pub fn name(self: Timetype) [:0]const u8 {
+    pub fn name(self: *Timetype) [:0]const u8 {
```

Comment:
Shouldn't this be `self: *const Timetype`, since it returns a const pointer?

Reply:
It would be better.